doc/ipython-notebooks/multiclass/Tree/DecisionTrees.ipynb

```diff
@@ -1405,9 +1405,9 @@
    "source": [
     "def train_chaidtree(dependent_var_type,feature_types,num_bins,feats,labels):\n",
     "    # create CHAID tree object\n",
-    "    c = sg.create_machine(\"CHAIDTree\", m_dependent_vartype=dependent_var_type,\n",
```

Comment:
perfect!

Reply:
I think you forgot some, I remember seeing one name in a "get" call...double check

Reply:
Are you referring to this:
https://github.com/shogun-toolbox/shogun/blob/123f5124b731751e079ab8f138c0808361a3bea6/doc/ipython-notebooks/multiclass/Tree/DecisionTrees.ipynb#L613

I don't think we should change this right? As it isn't related to CHAID tree? As then we will have to make some other change in the source code related to `C45ClassifierTree`

Also, the other one is :
https://github.com/shogun-toolbox/shogun/blob/123f5124b731751e079ab8f138c0808361a3bea6/doc/ipython-notebooks/multiclass/Tree/DecisionTrees.ipynb#L643

This one too, is an instance of `C45ClassifierTree`. Should I change these two instances too?

Reply:
ah sorry. of course you are right!
We will see it in the CI for the notebooks